staging/src/k8s.io/client-go/examples/in-cluster/main.go

```diff
@@ -43,6 +44,21 @@ func main() {
 			panic(err.Error())
 		}
 		fmt.Printf("There are %d pods in the cluster\n", len(pods.Items))
+
+		// Examples for error handling:
```

Comment:
I don't like that too. But I'm afraid that people won't look at both (in cluster / out of cluster) examples, but only at the one they need...?
Or should I create a new folder `examples/errorhandling`?